README.md

```diff
@@ -32,6 +32,37 @@ The repository has the following permanent branches:
 
  * **apk** This branch contains the apks for the code in the develop branch. The apks are automatically updated when a commit is pushed to `develop` branch.
 
+
+##Run Server Locally To Test Android
+
+ * **Run server on Local machine methods.**
+	1. Run by local ip :-
+		 Connect your mobile and laptop on the same network. Type command ipconfig if you are using Linux or ipconfig if you are using windows. Modify the run.py by following line 
+
+		if __name__ == "__main__":
+		application.debug=True
+		application.run(host='<local-ip>',port=5000)
+	        <local-ip> be like 192.168.xx.xx
```

Comment:
Do you really need this step? I just used localhost. Just let me know, if you need, perhaps because of ngrok  tool 🤷‍♀ I just want to understand

Reply:
We can do that also but it is easy to understand. Assigning same ip as of mobile.